README.md

```diff
@@ -1 +1,44 @@
-# math-magician
\ No newline at end of file
+# Math Magicians
+
+> Project for getting starting with React.
+
+
+## Built With
+
+- HTML, CSS, and JavaScript
+- React
+- git/GitHub
+
+### `Getting Started
+To get a local copy up and running follow these simple example steps.
+
+  - go to the directory that you want to clone the repository in.
+  - right click and choose Git Bash here.
+  - type "git clone https://github.com/Wes-Isaac/Math-Magician.git".
+  - run "npm install" and then "npm start"
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.